eda/src/autogluon/eda/visualization/layouts.py

```diff
@@ -0,0 +1,75 @@
+from typing import Union, List, Dict
+
+from ipywidgets import HBox, Output, Layout, Tab
+
+from .base import AbstractVisualization
+from .jupyter import JupyterMixin
+from .. import AnalysisState
+
+
+class SimpleVerticalLinearLayout(AbstractVisualization):
+
+    def __init__(self,
+                 facets: Union[AbstractVisualization, List[AbstractVisualization]],
+                 namespace: str = None,
+                 **kwargs) -> None:
+        super().__init__(namespace, **kwargs)
+        if not isinstance(facets, list):
+            facets = [facets]
+        self.facets = facets
+        self._kwargs = kwargs
+
+    def can_handle(self, state: AnalysisState) -> bool:
+        return True
+
+    def _render(self, state: AnalysisState) -> None:
+        for facet in self.facets:
+            facet.render(state)
+
+
+class SimpleHorizontalLayout(SimpleVerticalLinearLayout, JupyterMixin):
```

Comment:
A general comment for all these classes and might not be necessary, but I'm thinking maybe a link to a simple example visualize how the layout look like?